src/csv_reporter.cc

```diff
@@ -37,6 +37,17 @@ std::vector<std::string> elements = {
     "error_occurred", "error_message"};
 }  // namespace
 
+std::string CsvEscape(const std::string & s) {
+  std::string tmp;
```

Comment:
```suggestion
  std::string tmp;
  tmp.reserve(s.size() + 2);
```

Reply:
we should add up all the nanoseconds ever saved by this change and compare with the amount of time we spent arguing about it.